compiler/rustc_mir_build/src/build/mod.rs

```diff
@@ -993,13 +993,72 @@ fn parse_float_into_constval<'tcx>(
     parse_float_into_scalar(num, float_ty, neg).map(ConstValue::Scalar)
 }
 
+#[cfg(not(bootstrap))]
+fn parse_check_f16(num: &str, f: Half) -> Option<()> {
+    let Ok(rust_f) = num.parse::<f16>() else { return None };
+
+    assert!(
+        u128::from(rust_f.to_bits()) == f.to_bits(),
+        "apfloat::ieee::Half gave a different result for `{num}`: \
+         {f} ({:#x}) vs Rust's {} ({:#x})",
+        f.to_bits(),
+        Single::from_bits(rust_f.to_bits().into()),
+        rust_f.to_bits()
+    );
+
+    Some(())
+}
+
+// FIXME:f16_f128: bootstrap `f16` parsing via `f32`
```

Comment:
Fix this fixme

Reply:
Actually it should be fixed, I'll remove it.

After a beta, does somebody go through and adjust all the `#[cfg(bootstrap)]`s?

Reply:
> After a beta, does somebody go through and adjust all the #[cfg(bootstrap)]s?

Yes